packages/ui/src/containers/DeployContractsContainer.ts

```diff
@@ -411,8 +419,8 @@ export class DeployContractsContainer {
     const preState = localStorage.getItem(
       DeployContractsContainer.PersistentKey
     );
-    let restoreDeployArgument = function(arg: RawDeployArguments) {
-      let helper = function(
+    let restoreDeployArgument = function (arg: RawDeployArguments) {
```

Comment:
I don't think you should be using `function` keyword in TypeScript. Change it to `(args) => {}` notation.

Reply:
Sure thing